documents-d-archeologie-francaise.csl

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
sorry, this is actually the fault of our formatter tool, but could you lowercase utf-8 again, please?